src/main/java/com/microsoft/sqlserver/jdbc/SQLServerCallableStatement.java

```diff
@@ -487,7 +543,8 @@ public int getInt(int index) throws SQLServerException {
     public int getInt(String parameterName) throws SQLServerException {
         loggerExternal.entering(getClassNameLogging(), "getInt", parameterName);
         checkClosed();
-        Integer value = (Integer) getValue(findColumn(parameterName), JDBCType.INTEGER);
+        Integer value = (Integer) getValue(
+                findColumn(parameterName, CallableStatementGetterSetterMethod.isGetterMethod), JDBCType.INTEGER);
```

Comment:
why are we adding this everywhere when this param isn't even used in the method?